resources/lang/en-US/general.php

```diff
@@ -215,6 +215,8 @@
     'no_results'			=> 'No Results.',
     'no'  					=> 'No',
     'notes'  				=> 'Notes',
+    'note_added'            => 'Note Added',
```

Comment:
I don't think we need this translation after all.

Reply:
This translation isn't being used right? I think only `add_note` is.

Reply:
Ah...this is used for translating the action in the history tab 👍🏾

Reply:
we do use this translation in the action column on the history tab